tools/benchmark.py

```diff
@@ -154,6 +155,19 @@ def run_syscall_count_benchmark(deno_path):
     return syscall_count_map
 
 
+def find_max_mem(out):
```

Comment:
Does this return a value in megabytes? Can you add a comment that says that (or better, add megabyte to the function name so far... e.g. `extract_max_mem_in_megabtes`

Reply:
@ry done